src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/PackageRestoreDataSource.cs

```diff
@@ -62,20 +62,41 @@ internal class PackageRestoreDataSource : ChainedProjectValueDataSourceBase<Rest
         private readonly IFileSystem _fileSystem;
         private readonly IProjectDiagnosticOutputService _logger;
         private readonly IProjectDependentFileChangeNotificationService _projectDependentFileChangeNotificationService;
-
+        private readonly IVsSolutionRestoreService4 _solutionRestoreService4;
         private byte[]? _latestHash;
         private bool _enabled;
 
+        /// <summary>
+        /// Project Unique Name used by Nuget Nomination.
+        /// </summary>
+        public string Name => _project.FullPath;
```

Comment:
Move this down to the other properties.

Reply:
Done in [commit ](https://github.com/dotnet/project-system/pull/7199/commits/d5a2a1f9b385e6a96d6d1c5fcbde5b7d0f19ebc5)